docs/en/operations/configuration-files.md

```diff
@@ -55,6 +55,68 @@ $ cat /etc/clickhouse-server/users.d/alice.xml
 </yandex>
 ```
 
+## YAML examples {#example}
```

Comment:
Let's add a link to example in the repository (default config in YAML with comments).